pcsx2/CDVD/CdRom.cpp

```diff
@@ -926,10 +939,10 @@ void cdrWrite1(u8 rt)
 			break;
 
 		case CdlReadS:
-			if (cdvd.Type == CDVD_TYPE_PSCDDA) // Taken from pcsxr
-				goto do_CdlPlay;
 			cdr.Irq = 0;
 			StopReading();
+			if (cdvd.Type == CDVD_TYPE_PSCDDA) // Taken from pcsxr
```

Comment:
This is likely wrong. Again, the command doesn't care about the media/track type, how it gets delivered is (iirc) based on the subq of the sector itself.

Reply:
Yeeahhh. It was an old hack from pcsxr, I'll remove it